### ISSUE TITLE
WormMine hide method usage and instead use the SO_term attribute of the method

### ISSUE THREAD

**Issue by Paul-Davis** [WormBase/website#4556]
Recently there was a help query where the user was looking for all non_coding genes in c. elegans and was confused by the method types, as a result of this the curators feel that it's probably better to use the SO_term attribute in the method instead of the method name itself.

Example mine query
```
<query name="" model="genomic" view="Gene.primaryIdentifier Gene.secondaryIdentifier Gene.symbol Gene.transcripts.method" longDescription="" sortOrder="Gene.transcripts.method asc" constraintLogic="A and B">
  <constraint path="Gene.organism.name" code="A" op="=" value="Caenorhabditis elegans"/>
  <constraint path="Gene.transcripts.method" code="B" op="LIKE" value="*RNA*"/>
</query>
```

Here I'm using the "*RNA*" to constrain the transcript method, but it might be better to use the GFF_SO value and SO_name, as using a generic RNA method can bring in additional false positives.

Methods:
![rna_methods](https://cloud.githubusercontent.com/assets/717707/12833097/36eef7aa-cb53-11e5-804f-6df01ffff284.png)

SO_terms:
![rna_gff_so_term](https://cloud.githubusercontent.com/assets/717707/12833105/3fb16da0-cb53-11e5-9520-fa30299bad35.png)

Basically hide this and replace with the above.

![so_query_builder](https://cloud.githubusercontent.com/assets/717707/12833186/e2f82170-cb53-11e5-9a3e-fb8c5526ead1.png)
